tests/modules/notebooks/test_sagemaker_notebook.py

```diff
@@ -10,75 +20,35 @@ def org1(org, user, group, tenant):
 
 
 @pytest.fixture(scope='module')
-def env1(env, org1, user, group, tenant, module_mocker):
+def env1(env, org1, user, group, tenant, db, module_mocker):
```

Comment:
Why do we need `db`? @nikpodsh

Reply:
No, we don't need it. Thanks!

Reply:
- [x] Clean up of parameters in test method